examples/showcase/firefox.py

```diff
@@ -0,0 +1,63 @@
+import re
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.path import Path
+import matplotlib.patches as patches
+
+# From: http://raphaeljs.com/icons/#firefox
```

Comment:
This address redirects to the main page. And clicking on "Free Icons" gets me a 404 error. I see no information about licensing. I am almost certain that firefox icons from mozilla aren't compatible with BSD-like licenses (which is why the BSD OSes ship with rebranded Firefox).